src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_Synthesized.cs

```diff
@@ -652,6 +652,279 @@ internal class C2<[Attr] T2> { }
             Assert.Equal(new[] { "Attr" }, GetAttributeNames(typeParam.GetAttributes()));
         }
 
+        [Fact]
+        [WorkItem(46439, "https://github.com/dotnet/roslyn/issues/46439")]
+        public void RecordSynthesizedMembers()
```

Comment:
Do we have tests validating the behavior when the `CompilerGenerated` attribute is missing?

Reply:
@jaredpar I added the test, but I'm not sure if there is an existing test already somewhere.

Reply:
There seems to be a legitimate test failure with `AttributeTests_Synthesized.AttributeIsMissing`

Reply:
@jcouv I'm not familiar with PEVerify issues/failures. Should I skip verification or there is something else to do?